README.md

```diff
@@ -315,15 +329,19 @@ is omitted, then the output will be returned to stdout.  Here are the sign
 >```
 >#### Description
 >
->Return a signature request URL from a Digital Twin document.  The URL can be
->shared with other people to sign a document.  If the input parameter is
->omitted, then the command will take its input from stdin.  Returns to stdout
->the URL to be handed to the signer.
+>Create a document signature URL for the person with the given email address and name.
+>The signature request URL can either be sent to the user mailbox or returned by this
+>command. If the request URL is sent by email, then it is not returned by the command.
+>If the input parameter is omitted, then the command will take its input from stdin.
+>Returns to stdout the URL to be handed to the signer if the URL is not sent by email
+>otherwise return nothing.
 >#### Flags
 >```
->--name    string   Full name of signer
->--email   string   Email address of signer [required]
->--hash    string   The hash of a Digital twin document
+>--name         string   Full name of signer
+>--email        string   Email address of signer [required]
+>--hash         string   The hash of a Digital twin document
+>--requestonly  string   Do not prepare the document if it is not already prepared and return an error instead.
+>--send-email   bool     When this flag is enabled API will send signing request to the user.
```

Comment:
here dash is used to separate send and email requestonly and sigonly flag also concatenated

Reply:
edc52cdf46a63a8c617fcd251c4e4a13fc647095